OmgvaPOS/src/UserManagement/Controller/UserController.cs

```diff
@@ -30,7 +30,7 @@ public class UserController(IAuthenticationRepository authenticationRepository,
         // [ProducesResponseType(StatusCodes.Status401Unauthorized)] 
         [ProducesResponseType(StatusCodes.Status409Conflict)]
         [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-        public IActionResult SignIn([FromBody]SignInRequest signInRequest)
+        public IActionResult SignIn([FromBody]SignInRequest signInRequest) //sita reiks padaryti, kad tik admin ar owneriui butu butu
```

Comment:
But how? You cannot check the role while signing in

Reply:
We are making so that you can not create your own account. Therefore, the one who is creating it has to be logged in.

Reply:
We decided that the first account will be created manually via database query, and other can be added via our API